### PR TITLE
Adds Ash Plants

### PR DESCRIPTION
**Pull request by ChangelingRain**
Adds a bunch of ash plants.
yeah it's this again

TODO convince kor to use these somewhere
TODO make soup sprites for delicious mushroom soup in not so delicious mushroom bowl
TODO make leaves require springs to wash instead of watery ground once springs exist

### CODE REVIEW

**Comment by TechnoAlchemisto** [comment]
a l c h e m y ?

**Comment by Iamgoofball** [comment]
you know this is greyscaled

we could have No Man's Sky

**Comment by ChangelingRain** [comment]
these aren't greyscaled

**Comment by KorPhaeron** [comment]
technoalchemy

@ChangelingRain 

Just add them on to the pile of nonsense that is the tunnel spawning code obviously

**Comment by ChangelingRain** [comment]
aaaaaaaaaaa A: NO THESE ARE FOR ASH PLANET NOT LAVALAND B: AAAAAAAAAA TUNNEL SPAWNING CODE IS HELL

**Comment by Kromgar** [comment]
Ash planet?

**Comment by Dorsisdwarf** [comment]
the surface of lavaland, I think. A bunch of sprites were done for it but I guess kor didnt want to work on it?

**Comment by ChangelingRain** [comment]
Yeah it'd have been the surface but kor never did it and I'm jabbing at him to maybe finish it

**Comment by Kromgar** [comment]
Wait so your saying lavaland would be the caverns underneath ash planet?

**Comment by Dorsisdwarf** [comment]
Yes, that's why it's 99% solid rock with massive, black pits, no light, and lava everywhere (Ash storm fluff doesn't respect it tho)

File: code/modules/mining/lavaland/ash_flora.dm [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+	desc = initial(desc)
+	harvested = FALSE
+
+/obj/structure/flora/ash/attackby(obj/item/weapon/W, mob/user, params)
+	if(!harvested && needs_sharp_harvest && W.sharpness)
+		user.visible_message("<span class='notice'>[user] starts to harvest from [src] with [W].</span>","<span class='notice'>You begin to harvest from [src] with [W].</span>")
+		if(do_after(user, harvest_time, target = src))
+			harvest(user)
+	else
+		..()
```
Comment (phil235) [inline code/modules/mining/lavaland/ash_flora.dm]: return ..()

**Comment by Kromgar** [comment]
So what kind of fauna would be on the ash planet? Should be like magical criters everything si friendly. Descend into lavaland its all hellish. Go through necropolis oh mah gawd its... heaven?

Seriously really bothered me you cant go through the necropolis door

**Comment by Chocobro** [comment]
:+1:
consider some kind of plant transplanting procedure
(a pot)

**Comment by nullbear** [comment]
some plants should kill you just for walking over them. Not right away though, of course.

'walks over innocent looking plant -> starts to feel drowsy after a minute or so -> starts dying about a minute later.'

Or alternatively start tripping balls when you accidentially bump into one.

**Pull request merged**